lab-andrew/gulpfile.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+
+
+gulp.task('test', function() {
+  return gulp.src(__dirname + '/test/*.js')
```

Comment:
This could be refactored to just say 'test/*.js' instead of __dirname + '/test/*.js'. :)